include/libtorrent/hash_picker.hpp

```diff
@@ -160,7 +160,7 @@ namespace libtorrent
 
 		struct piece_hash_request
 		{
-			time_point last_request;
+			time_point last_request = min_time();
```

Comment:
this would prevent the hash_picker from sending hash requests in the simulations (probably because the simulated clock starts too low). Technically, this is more correct though